_posts/2018-06-05-Rust-1.26.2.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "Announcing Rust 1.26.2"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce a new version of Rust, 1.26.2. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed via rustup, getting Rust
+1.26.2 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.26.2][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1262-2018-06-05
+
+## What's in 1.26.2 stable
+
+An unsoundness in the borrow checker verification of match statements was found.
```

Comment:
I think we should call out that this is a bug in the just-stabilized changes to match that first appeared in 1.26.

Reply:
How does that sound?